base/interactiveutil.jl

```diff
@@ -340,11 +340,11 @@ function versioninfo(io::IO=STDOUT; verbose::Bool=false, packages::Bool=false)
     end
     if packages || verbose
         println(io, "Packages:")
-        println(io, "  Package Directory: ", Pkg.dir())
+        println(io, "  Package Directory: ", PKG_MODULE_REF[].dir())
         print(io, "  Package Status:")
-        if isdir(Pkg.dir())
+        if isdir(PKG_MODULE_REF[].dir())
```

Comment:
I think you might need to use `invokelatest` here. @vtjnash should be able to confirm that.

Reply:
Seems to work ok actually.